src/main/java/mate/academy/spring/validation/EmailValidator.java

```diff
@@ -0,0 +1,20 @@
+package mate.academy.spring.validation;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+
+public class EmailValidator implements ConstraintValidator<Email, String> {
+    public static final String EMAIL_PATTERN = "^(.+)@(.+)$";
+
+    @Override
+    public boolean isValid(String email, ConstraintValidatorContext constraintValidatorContext) {
+        if (email == null) {
+            return false;
+        }
+        Pattern pattern = Pattern.compile(EMAIL_PATTERN);

```

Comment:
![image](https://user-images.githubusercontent.com/20326858/229488783-2bae20bd-fc95-4f4d-8ae2-2457de5c24f2.png)

Reply:
oh yes,it made my code better